packages/web/src/features/landing/frames/NoticeSectionFrame.tsx

```diff
@@ -0,0 +1,49 @@
+"use client";
+
+import React from "react";
+import styled from "styled-components";
+import { NoticeInfo } from "@sparcs-clubs/web/features/notice/types/notice.type";
+import Link from "next/link";
+import paths from "@sparcs-clubs/web/constants/paths";
+import MoreSectionTitle from "../components/MoreSectionTitle";
+import NoticeCard from "../components/NoticeCard";
+
+const NoticeSectionFrameInner = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  width: 70%;
```

Comment:
영역이 70% 30%로 나뉘는 게 아니구 나머지랑 320px로 나뉩니당

```suggestion
  gap: 16px;
  overflow: hidden;
```